src/configurations/adknowledgeportal/routesConfig.ts

```diff
@@ -239,33 +240,35 @@ const routes: GenericRoute[] = [
     ],
   },
   {
-    to: 'Explore',
-    isNested: true,
+    path: 'Explore',
     routes: [
```

Comment:
In the type definitions, we now enforce that
* A route can only have a `synapseConfigArray` or child `routes`, but not both
* If a route has `routes`, `exact` is implicitly false, and cannot be overridden.

With how the new RouteResolver works, I can't think of a reason where you would want `exact` to be true and render child routes, because then the child routes could never render.